bloggies_frontend/src/Forms/BlogForm/index.tsx

```diff
@@ -14,17 +14,21 @@ interface IProp {
  * `BlogForm` renders a form for creating/editting a blog.
  * (IF EDITING: post and closeModal parameters required)
  */
-function BlogForm({addPost, post, closeModal }: IProp) {
+function BlogForm({ addPost, post, closeModal }: IProp) {
   const membershipStatus = useSelector(
     (st: CustomReduxState) => st.user.membership_status
   );
-  const INITIAL_FORM_VALUES = { title: post?.title || "", description: post?.description || "", body: post?.body || "", is_premium: membershipStatus === 'active'}
+  const INITIAL_FORM_VALUES = { title: post?.title || "", description: post?.description || "", body: post?.body || "", is_premium: false };
```

Comment:
For editing the blog, can we do a check for post.is_premium so the box will automatically be checked if it's a premium post?

Reply:
done!